MonoGame.Framework/Graphics/RenderTarget2D.cs

```diff
@@ -24,9 +24,29 @@ private bool SuppressEventHandlerWarningsUntilEventsAreProperlyImplemented()
         }
 
 	    public RenderTarget2D(GraphicsDevice graphicsDevice, int width, int height, bool mipMap, SurfaceFormat preferredFormat, DepthFormat preferredDepthFormat, int preferredMultiSampleCount, RenderTargetUsage usage, bool shared, int arraySize)
-	        : this(graphicsDevice, width, height, mipMap, preferredFormat, preferredDepthFormat, preferredMultiSampleCount, usage, SurfaceType.RenderTarget, shared, arraySize)
+	        : base(graphicsDevice, width, height, mipMap, QuerySelectedFormat(graphicsDevice, preferredFormat), SurfaceType.RenderTarget, shared, arraySize)
 	    {
+            DepthStencilFormat = preferredDepthFormat;
+            MultiSampleCount = graphicsDevice.GetClampedMultisampleCount(preferredMultiSampleCount);
```

Comment:
Call to GetClampedMultisampleCount() is also restored here.
This change affected all platforms.